src/Storefront/Resources/app/storefront/src/plugin/main-menu/offcanvas-menu.plugin.js

```diff
@@ -10,7 +10,7 @@ export default class OffcanvasMenuPlugin extends Plugin {
     static options = {
         navigationUrl: window.router['frontend.menu.offcanvas'],
         position: 'left',
-        tiggerEvent: 'click',
+        triggerEvent: 'click',
```

Comment:
I think we have to deprecate the old option, instead of removing it. since it could be in use already.
we also have to make sure, that the old option still works as well

Reply:
Hi @PheysX, @mitelg is right, we should deprecate the option.

Would you mind to add the deprecation for v.6.4.0, so both the old and the new option are still working ?